Protos/V1/sessions_service.proto

```diff
@@ -0,0 +1,157 @@
+syntax = "proto3";
+
+import "objects.proto";
+import "session_status.proto";  
+
+option csharp_namespace = "ArmoniK.Api.gRPC.V1.Sessions";
+package ArmoniK.api.grpc.V1.sessions;
+
+/**
+ * The SessionService provides methods for listing, getting and cancelling sessions
+ */
+service Sessions {
+  // Get all sessions with pagination
+  rpc ListSessions(ListSessionsRequest) returns (ListSessionsResponse);
+
+  // Get a session using an ID
+  rpc GetSession(GetSessionRequest) returns (GetSessionResponse) {}
+
+  // Cancel a session using an ID
+  rpc CancelSession(CancelSessionRequest) returns (CancelSessionResponse) {}
+}
+
+/**
+ * A raw session object
+ */
+message SessionRaw {
+  string session_id = 1;
+  SessionStatus.SessionStatus status = 2;
+  TaskOptions.TaskOptions options = 3;
+
+  string created_at = 4;
+  string cancelled_at = 5;
+}
+
+/**
+ * A session object
+ */
+message SessionSummary {
+  string session_id = 1;
+
+  int64 count_tasks = 2;
+  int64 count_tasks_pending = 3;
+  int64 count_tasks_error = 4;
+  int64 count_tasks_completed = 5;
+  int64 count_tasks_processing = 6;
+
+  SessionStatus.SessionStatus status = 7;
+  string created_at = 8;
```

Comment:
Date object

Reply:
https://stackoverflow.com/questions/44468734/import-timestamp-in-proto-file-of-protobuf-for-grpc

Reply:
Yes, merci pour le lien !